editor/components/post-publish-button/label.js

```diff
@@ -37,6 +37,8 @@ export function PublishButtonLabel( {
 		return __( 'Publishing…' );
 	} else if ( isPublished && isSaving ) {
 		return __( 'Updating…' );
+	} else if ( isBeingScheduled && isSaving ) {
```

Comment:
There are very simple corresponding tests which help to avoid regressions. Can we cover this case, too?